plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java

```diff
@@ -13,7 +13,6 @@
  */
 package io.trino.plugin.iceberg;
 
```

Comment:
The PR and commit description needs an update. There isn't any real problem with orc/parquet statistics here, the scenario described has wrong statistics which we shouldn't be changing anything for. Now the PR is about improving split pruning on worker based on dynamic filters.